charts/aws-vpc-cni/crds/customresourcedefinition.yaml

```diff
@@ -1,10 +1,9 @@
-{{- if .Values.crd.create -}}
```

Comment:
Can you remove this field from `values.yaml` file? Once this is removed, I can approve